kong/clustering/control_plane.lua

```diff
@@ -248,6 +249,9 @@ end
 
 
 function _M:push_config()
+  ngx_update_time()
```

Comment:
I think this can be removed without affecting the result (we can keep the later one as that is after the CPU intensive port of the export).

Reply:
Thanks!
```suggestion
```